app/controllers/action_officers_controller.rb

```diff
@@ -2,11 +2,18 @@ class ActionOfficersController < ApplicationController
   before_action :authenticate_user!, PQUserFilter
 
   def index
-    @action_officers = ActionOfficer.active_list
-                                    .joins(deputy_director: :division)
-                                    .order(deleted: :asc)
-                                    .order(Arel.sql('lower(divisions.name)'))
-                                    .order(Arel.sql('lower(action_officers.name)'))
+    @show_inactive = (params[:show_inactive] == 'true')
+    @action_officers = if @show_inactive
```

Comment:
I'd be tempted to do something like
`list = @show_inactive ? ActionOfficer.inactive_list : ActionOfficer.active_list`

instead of re-using `@action_officers`

Reply:
This makes the code simpler and changed as above